pkg/machines/components/nicEdit.jsx

```diff
@@ -53,10 +53,26 @@ class EditNICModal extends React.Component {
     constructor(props) {
         super(props);
 
+        let defaultNetworkSource;
+        let currentSource;
+        let availableSources = [];
+        if (props.network.type === "network") {
+            availableSources = props.networks.map(network => network.name);
+            currentSource = props.network.source.network;
+        } else if (props.network.type === "direct") {
```

Comment:
Since https://github.com/cockpit-project/cockpit/pull/12780 got merge, this condition should also include a case when `props.network.type === "bridge"`

Reply:
Done.